views/update/review-update-statement.html

```diff
@@ -20,8 +20,6 @@
 
 {% set noChange %}
   {% if appData.update.no_change == 1 %}
-    Yes
```

Comment:
Just for other reviewer's awareness - Matthew confirmed it is not possible to get to this page and have "yes" as an option.